Automattic/jetpack
# Social Logos: update to include a Fediverse and a Nextdoor logo.

Fixes #14280 

## Proposed changes:

This PR updates our social logos icon font and introduces new logos for our Social Icons widget and Social Menu theme tool, matching the Social Logos update, 2.5.3:

* https://github.com/Automattic/social-logos/pull/119
* https://github.com/Automattic/social-logos/pull/120

It also introduces a new filter, that was asked in #14280 and that will make testing this PR easier.

Related:

- #7389 
- #30675 

> **Note**
> This doesn't introduce any frontend elements by default for the Fediverse logo yet, since there isn't a single domain name we can use to point to a Fediverse site.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to `yoursite.com/wp-content/plugins/jetpack/_inc/social-logos/social-logos.html`
* Check the new logos
* Go to Jetpack > Settings and enable Sharing buttons as well as the extra sidebar widgets.
* Switch to an old theme, like Twenty Ten.
* Go to Plugins > Add New and install the Classic Widgets plugin.
* In a functionality plugin, add the following:
```php
// Add social menu support
add_action( 'after_setup_theme', function () {
	add_theme_support( 'jetpack-social-menu', 'svg' );
} );
add_filter(
	'jetpack_social_icons_supported_icons',
	function ( $social_links_icons ) {
		$social_links_icons[] = array(
			'url'   => array( 'fedi.jeremy.hu' ),
			'icon'  => 'fediverse',
			'label' => 'Fediverse',
		);
		return $social_links_icons;
	}
);
```
* Edit your theme and add the following, in `footer.php` for example
```php
<?php jetpack_social_menu(); ?>
```
* Go to Appearance > Customize > Menus
* Create a new menu with some links, e.g. `https://www.facebook.com/jetpackme` and `https://nextdoor.com/city/agua-dulce--ca/`
* Assign that menu to the Social Menu location
* Go to Appearance > Widgets menu
* Drag the Social Icons widget to one of the active widget areas, and add some links again. Add a custom URL using this URL too: `https://fedi.jeremy.hu/@jeremy`
* Check how things look like on the frontend, for both your social menu and the Social Icons Widget you created.
